torch/nn/functional.py

```diff
@@ -3766,6 +3766,15 @@ def upsample(input, size=None, scale_factor=None, mode="nearest", align_corners=
     upsample.__doc__ = upsample.__doc__.format(**reproducibility_notes)
 
 
+def is_integer(x) -> bool:
```

Comment:
You don't want to make that function public: it should start with `_*`.

Reply:
```suggestion
def _is_integer(x) -> bool:
```